src/converters/light.ts

```diff
@@ -29,7 +29,21 @@ import {
 import { convertHueSatToXy, convertMiredColorTemperatureToHueSat, convertXyToHueSat } from '../colorhelper';
 import { EXP_COLOR_MODE } from '../experimental';
 
+interface LightConfig {
+  adaptive_lightning_enabled?: boolean;
```

Comment:
It has a typo and I would remove the `_enabled` suffix.
If we need more features in the future, we can possibly make this into an `boolean | object` instead of just a boolean.

```suggestion
  adaptive_lighting?: boolean;
```